File: cinema-frontend/src/app/user/dtos/response/cinema-manager-table.response.ts
import {CinemaResponse} from "../../../cinema/dtos/response/cinema.response";

export class CinemaManagerTableResponse {
  constructor(
    readonly firstName: string,
    readonly lastName: string,
    readonly email: string,
    readonly managedCinema: CinemaResponse,
  ) {
  }
}

File: cinema-backend/src/main/java/org/example/cinemabackend/user/application/dto/response/CinemaManagerTableResponse.java
package org.example.cinemabackend.user.application.dto.response;

import lombok.Builder;
import org.example.cinemabackend.cinema.application.dto.response.CinemaResponse;

@Builder
public record CinemaManagerTableResponse(
        String firstName,
        String lastName,
        String email,
        CinemaResponse managedCinema
) {
}
